### ISSUE TITLE
A Comparison of Deep Learning Object Detection Models for Satellite Imagery

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#121]
# TL;DR
A study comparing object detection models using satellite images. The single-stage model works fast and the accuracy is not as bad. However, the accuracy for detecting small objects, such as cars, was low. If we hyperparameters-tuned it for small objects, we might be able to get a better performance with a single stage model, they said.
![ResNet-101](https://user-images.githubusercontent.com/24524018/93829855-97857580-fca9-11ea-9eb3-f18d7e2efd3f.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2009.04857

# Submission Dates(yyyy/mm/dd)
2020/09/10

# Authors and institutions
Austen Groener, Gary Chern, Mark Pritt
- Lockheed Martin Space

# Methods

# Results

# Comments
